### ISSUE TITLE
Remove block comment not working in Spyder 3.3.0

### ISSUE THREAD

**Issue by iamjmat** [spyder-ide/spyder#7676]
Not able to remove block comment in Spyder 3.3.0. 
After selecting the necessary lines, Edit-->Remove Block Comment doesnt seem to work.

I installed spyder from pip and here are the dependencies
Python 3.6.3 64bits, Qt 5.9.3, PyQt5 5.9.2 on Windows

**Comment by ccordoba12** [comment]
Thanks for reporting.

@CAM-Gerlach, are you able to reproduce this one?

**Comment by CAM-Gerlach** [comment]
@ccordoba12 Adding and removing a block comment worked fine for me, as long as I selected the lines *inside* the blog comment rather than including the comment lines. @iamjmat can you give an example code snippit where this happens and indicate exactly what was selected when you executed the command?

Also, not directly related, but we need to remove the first ``=``to the right of the ``#`` on each added line to follow PEP and avoid warnings by our own ``pycodestyle`` real-time checker.

**Comment by ccordoba12** [comment]
> Also, not directly related, but we need to remove the first =to the right of the # on each added line to follow PEP

I think this is fixed in master. Could you try it, please?

**Comment by CAM-Gerlach** [comment]
@ccordoba12 Yup, it is fixed. Thanks!

**Comment by iamjmat** [comment]
@CAM-Gerlach Yeah. It works for me when I select the lines inside the blog comment.
Is it possible to make it work if we select the `=` lines too ?

**Comment by CAM-Gerlach** [comment]
Its possible, I suppose, but likely somewhat more complicated. If you'd like to implement it, unless @ccordoba12 says otherwise you're welcome to send us a PR, since Spyder is written in the same language (Python) as you write in it (in fact, Spyder is the primary tool used to develop Spyder itself).

**Comment by iamjmat** [comment]
Cool. Will try it out.
Learning Python for a few months and Spyder is by far the best IDE out there. Hats off to everyone for creating such a really good IDE.
